apps/src/music/blockly/blocks/simple2.js

```diff
@@ -105,6 +105,7 @@ export const triggeredAtSimple2 = {
     style: 'event_blocks',
     tooltip: musicI18n.blockly_blockTriggeredTooltip(),
     extensions: [DYNAMIC_TRIGGER_EXTENSION],
+    helpUrl: '/docs/ide/projectbeats/expressions/trigger',
```

Comment:
Should we factor out the common path (`/docs/ide/projectbeats/expressions/`), into a variable or with the use of a helper function?

Reply:
Hmm yeah good call